php_phongo.c

```diff
@@ -1450,42 +1502,104 @@ static mongoc_client_t *php_phongo_make_mongo_client(const mongoc_uri_t *uri, zv
 		return NULL;
 	}
 
-	if (mongoc_uri_get_ssl(uri) && driverOptions) {
-		if (!php_phongo_apply_ssl_opts(client, driverOptions TSRMLS_CC)) {
-			mongoc_client_destroy(client);
-			return NULL;
-		}
+	if (mongoc_uri_get_ssl(uri) && ssl_opt) {
+		mongoc_client_set_ssl_opts(client, ssl_opt);
```

Comment:
I wanted to keep the freeing of `ssl_opt` local to the same function where it was allocated (i.e. `phongo_manager_init()`) for readability.